components/server/src/user/user-controller.ts

```diff
@@ -358,12 +359,16 @@ export class UserController {
                     return;
                 }
 
+                const useSameSiteNone = await getExperimentsClientForBackend().getValueAsync("sameSiteNone", false, {
```

Comment:
What is the goal of introducing a feature flag here? I'd like to understand the motivation and broader idea before we merge sth like this. :slightly_smiling_face: 
E.g., if the problem is "public port" is to public, then this is not a fix, but makes it worse. :shrug:

Reply:
I added a feature flag so we can disable it in case we overlooked something during testing